ui/imports/shared/views/TokenListView.qml

```diff
@@ -52,15 +54,11 @@ Rectangle {
                 }
             ]
         }
-        delegate: TokenDelegate {
-            objectName: "SendModal_AssetView_TokenListItem_" + symbol
-            readonly property string balance: enabledNetworkBalance // Needed for the tests
+        delegate: TokenBalancePerChainDelegate {
+            width: comboBox.control.popup.width
```

Comment:
Can tell where is this `comboBox` coming from

Reply:
hey it was a mistake I have already corrected it